tensorflow/core/transforms/remapper/remapping_helper.h

```diff
@@ -0,0 +1,260 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#ifndef TENSORFLOW_CORE_TRANSFORMS_REMAPPER_REMAPPING_HELPER_H_
+#define TENSORFLOW_CORE_TRANSFORMS_REMAPPER_REMAPPING_HELPER_H_
+
+#include "tensorflow/core/transforms/utils/op_cat_helper.h"
+#include "tensorflow/core/transforms/utils/utils.h"
+
+namespace mlir {
+namespace tfg {
+namespace remapping {
+
```

Comment:
Added comment. Please check.

Reply:
From what it's used, I think it flatens an operation. For example, in ContractionBiasAdd, the `contraction` is actually the operand of bias_add. Can't we just use the `bias_add`? I'm not sure how much these structures help because it seems that we don't have too much cases here, right?

Reply:
Since the logic in current grappler-remapper is bit convoluted, we thought that in the first stage,  we will follow mostly same logic. After we achieve identical functionality of the current grappler-remapper we will drop any redundancy or unoptimized codes. Please let me know what do you think?

Reply:
Got it, that makes sense to me. How about let's add a TODO here?

Reply:
Thanks! Added TODO.